archive-index.html

```diff
@@ -0,0 +1,897 @@
+                          <span aria-hidden="true">&times;</span>
+                        </button>
+                        <h2 class="modal-title" id="myModalLabel">Caged Bird</h2>
+                      </div>
+                      <div class="modal-body">
+                        A free bird leaps <br>
+                        on the back of the wind   <br>
+                        and floats downstream   <br>
+                        till the current ends <br>
+                        and dips his wing <br>
+                        in the orange sun rays <br>
+                        and dares to claim the sky. <br>
+                        <br>
+                        But a bird that stalks <br>
+                        down his narrow cage <br>
+                        can seldom see through <br>
+                        his bars of rage <br>
+                        his wings are clipped and   <br>
+                        his feet are tied <br>
+                        so he opens his throat to sing. <br>
+                        <br>
+                        The caged bird sings   <br>
+                        with a fearful trill   <br>
+                        of things unknown   <br>
+                        but longed for still   <br>
+                        and his tune is heard   <br>
+                        on the distant hill   <br>
+                        for the caged bird   <br>
+                        sings of freedom. <br>
+                        <br>
+                        The free bird thinks of another breeze <br>
+                        and the trade winds soft through the sighing trees <br>
+                        and the fat worms waiting on a dawn bright lawn <br>
+                        and he names the sky his own <br>
+                        <br>
+                        But a caged bird stands on the grave of dreams   <br>
+                        his shadow shouts on a nightmare scream   <br>
+                        his wings are clipped and his feet are tied   <br>
+                        so he opens his throat to sing. <br>
+                        <br>
+                        The caged bird sings   <br>
+                        with a fearful trill   <br>
+                        of things unknown   <br>
+                        but longed for still   <br>
+                        and his tune is heard   <br>
+                        on the distant hill   <br>
+                        for the caged bird   <br>
+                        sings of freedom.<br>
+                      </div>
+                      <div class="modal-footer">
+                        <p class="text-muted">
+                          Maya Angelou, “Caged Bird” from Shaker, Why Don't You Sing? Copyright © 1983 by Maya Angelou.
+                        </p>
+                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
```

Comment:
Between modal and share buttons we would add a "listen" button and link to each corresponding poem media file in the media folder.